core/domain.go

```diff
@@ -3,19 +3,18 @@ package lucy
 type DomainType uint
 
```

Comment:
Again, using enums `iota` would reduce the effort of writing incremental numbers yourself